src/main/java/qna/domain/Answers.java

```diff
@@ -0,0 +1,48 @@
+package qna.domain;
+
+import qna.CannotDeleteException;
+
+import javax.persistence.Embeddable;
+import javax.persistence.OneToMany;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+@Embeddable
+public class Answers {
+    @OneToMany(mappedBy = "question")
+    private final List<Answer> values = new ArrayList<>();
+
+    public void add(Answer answer) {
+        this.values.add(answer);
+    }
+
+    public boolean contains(Answer answer) {
+        return this.values.contains(answer);
+    }
+
+    public List<DeleteHistory> delete(User loginUser) throws CannotDeleteException {
+        List<DeleteHistory> deleteHistories = new ArrayList<>();
+        for (Answer answer : values) {
+            deleteHistories.add(answer.delete(loginUser));
+        }
+        return deleteHistories;
```

Comment:
stream을 사용해보는건 어떨까요??

```suggestion
        return answers.stream()
                .map(answer -> answer.delete(loginUser))
                .collect(toList());
```